### ISSUE TITLE
Cannot install Microsoft.VisualStudio.Debugger.Interop.14.0 through NuGet

### ISSUE THREAD

**Issue by etairi** [NuGet/Home#1861]
I tried to install the package `Microsoft.VisualStudio.Debugger.Interop.14.0` using NuGet (I tried both the command-line and UI version), but I get the following error messages, when it tries to install the dependency `Microsoft.VisualStudio.Debugger.InteropA`:

```
Install failed. Rolling back...
Package 'Microsoft.VisualStudio.Debugger.InteropA.9.0.21022 : ' does not exist in project ''
Package 'Microsoft.VisualStudio.Debugger.InteropA.9.0.21022 : ' does not exist in folder '\packages'
Install-Package : Could not install package 'Microsoft.VisualStudio.Debugger.InteropA 9.0.21022'. You are trying to install this package into a project that targets '.NETFramework,Version=v4.5.2', but the package does not contain any assembly references 
or content files that are compatible with that framework. For more information, contact the package author.
```

I asked some Microsoft guys, and they told me that the InteropA package has a single assembly directly in the lib folder, which NuGet is supposed to interpret as if it was under lib\net40, which certainly would be compatible with my Net452 project. But, unfortunately I get the error messages above when I try to install it. I also tried to install it into a project that targets Net46, but still no luck.

**Comment by yishaigalatzer** [comment]
I just tried doing this with both visual Studio 2013, and 2015 (Nuget 3.3), and both install fine. 

Can you please make sure you have the latest version of NuGet installed, and try again?

I'm closing as no-repro, but if you still can't resolve it, please post more details and a sample project where this doesn't work.

**Comment by etairi** [comment]
I'm using Visual Studio 2015 Enterprise Update 1, and I have NuGet 3.3. I cannot install that package in any kind of project or framework, neither in a VSIX project nor in a simple console application. Here is an example, where I created just a new console application, it is empty, when I tried to install the package, I got the following [output](http://prntscr.com/9hnzb9). I also tried to change the target framework to lower or higher .NET version, but still I get the same error messages. On the other hand, I can successfully install another VSSDK package, for example [this](http://prntscr.com/9ho2oh).
